instant-distance/src/lib.rs

```diff
@@ -130,7 +130,7 @@ impl Default for Heuristic {
 #[cfg_attr(feature = "serde", derive(Deserialize, Serialize))]
 pub struct HnswMap<P, V> {
     hnsw: Hnsw<P>,
-    values: Vec<V>,
+    pub values: Vec<V>,
```

Comment:
This didn't feel right, but I just punched through for simplicity for now.

Reply:
Check the develop branch for my latest API experiments.